R/download_ts.R

```diff
@@ -27,7 +27,7 @@
 #'   on_local_exists="skip")
 #' }
 #' @export
-download_ts <- function(var_src, site_name, folder = tempdir(), version=c('tsv','RData'),
+download_ts <- function(var_src, site_name, folder = tempdir(), version=c('tsv','rds','RData'),
```

Comment:
will change. Wasn't sure if we had any up there, and since download is really just a get file, I figured I would leave it. Will remove.